### DISCUSSION TITLE
Отображение поля из нескольких строк

### DISCUSSION THREAD

**Discussion by Leo-ui-528** [totumonline/totum-ru-issues-and-discussions#309] [Как это сделать?] [answered]
Здравствуйте, подскажите пожалуйста, как сделать поле в футере, чтобы сразу из нескольких строк по высоте, и все сразу отображались? Как текстовое, но сразу в таблице?

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Смотрите setFloatFormat и Секции
